src/test/java/org/jenkinsci/plugins/mesos/JenkinsSchedulerTest.java

```diff
@@ -249,7 +249,9 @@ public void testConstructMesosCommandInfoWithNullCustomDockerShell() throws Exce
                 null,               // externalContainerInfo,
                 containerInfo,      // containerInfo,
                 null);              //additionalURIs
-        Mesos.SlaveRequest slaveReq = new Mesos.SlaveRequest(new Mesos.JenkinsSlave(TEST_JENKINS_SLAVE_NAME),0.2d,TEST_JENKINS_SLAVE_MEM,mesosSlaveInfo);
+        Mesos.SlaveRequest slaveReq = new Mesos.SlaveRequest(
+                new Mesos.JenkinsSlave(TEST_JENKINS_SLAVE_NAME),0.2d,
```

Comment:
can you put each argument on its own line? we do this when we have to wrap a statement.